src/toast/toast.js

```diff
@@ -161,6 +161,7 @@ class Toast extends React.Component<ToastPropsT, ToastPrivateStateT> {
           <Body
             tabIndex={0}
             role="alert"
+            data-baseweb={this.props['data-baseweb'] || 'toast'}
```

Comment:
Also here, why do we want to get this value from props?

Reply:
Oh I see, these all are added for the reused internally components, correct? So we don't expose it still...

Reply:
Yep. For example, `notification -> toast`, `textarea->input` ...